tests/test_registrations/test_embargoes.py

```diff
@@ -515,6 +515,286 @@ def test_disapproval_cancels_embargo_on_descendant_nodes(self):
             assert_false(node.embargo_end_date)
 
 
+@pytest.mark.enable_bookmark_creation
+class LegacyRegistrationEmbargoApprovalDisapprovalViewsTestCase(OsfTestCase):
```

Comment:
I didn't restore all the old tests - just one set of them. This seemed sufficient to test that old links still work and we should remove these tests when the decorator is removed.